### DISCUSSION TITLE
Ethereal Port Design Doc

### DISCUSSION THREAD

**Discussion by KittyNoodle** [ParadiseSS13/Paradise#18164] [Documents]
### The Basics
Ethereals are a race of glowing crystal people who consume electricity to live, they prefer hotter environments and take increased brute damage.

### Nutrition System
Ethereals consume electricity instead of food, and are poisoned by regular food.
Ethereals gain electricity by clicking a light on help intent or an apc on grab intent.
Ethereals can also gain electricity from cyborg chargers.
Ethereals cannot be fat.

Ethereals take toxin damage when very low on charge instead of hypoglycemia.

### Light Mechanics

Ethereals have a selected light color similar to slimepeople's selected color
Ethereals' light color will shift towards an unhealthy hue and lower in strength when they are low on health.
When emped ethereals loose their light
When emagged ethereals will start flashing random colors for 2 minutes then return to their old color.

### Misc mechanics
Ethereals have a higher temperature preference and body temperture.
Ethereals take increased brute damage and decreased burn damage.
Ethereals do burn damage instead of brute when they punch.

**Comment by Sinestia** [comment]
What do you think of the crystallization mechanic?


 I think it reviving you is overpowered, but I think it could be worth keeping as a sort of stasis that minimizes organ decay as a perk upon death.
